File: JAVA PROGRAMMING/Test69.java
// 69. PERFECT SQUARE NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Num Value: ");
        int num = in.nextInt();

        int sqrtnum = (int)(Math.sqrt(num));
        boolean flag = (sqrtnum * sqrtnum == num);
        if(flag) {
            System.out.println(num+" Is Perfect Square Number.");
        } else {
            System.out.println(num+" Is Not Perfect Number.");
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test69.ts
// 69. PERFECT SQUARE NUMBER
const perfectSquare = (num: number): void => {
    let sqrtnum: number = Math.floor(Math.sqrt(num));
    let flag: boolean = (sqrtnum * sqrtnum === num);
    let res: string = (flag) ? `${num} Is A Perfect Square Number.` :
        `${num} Is Not A Perfect Number.`;
    console.log(res);
}
perfectSquare(36);
perfectSquare(15);